maintainers/team-list.nix

```diff
@@ -858,6 +858,18 @@ with lib.maintainers;
     shortName = "Vim/Neovim";
   };
 
+  nextcloud = {
+    members = [
+      bachp
+      britter
+      dotlambda
+      ma27
+    ];
```

Comment:
@provokateurin you started quite recently doing Nextcloud work here as well. Given this and you knowing the codebase way better than we do as developer, you'd be very welcome as member here. Let me know if you're interested and I'd add you.

Reply:
Sure! :heart:

Reply:
@provokateurin I just realized you don't have a maintainer entry in `maintainers/maintainer-list.nix` yet.

Would you mind filing a PR when this is merged where you add yourself there and in this team (feel free to cc me on this). I don't want to decide for you e.g. which email is in there, whether or not you want to provide a name etc..

Reply:
Yes, I'll do that :)